src/app/redux/reducers/ethrdid.ts

```diff
@@ -0,0 +1,55 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+import { DIDDocument } from 'did-resolver'
+
+export interface EtherdidState {
+  owner: string | null,
```

Comment:
The owner is inside the document

Reply:
I don't understand what you are saying here. The owner is inside of the DIDDocument? So there isn't a need to do lookupOwner? If this is true, which variable in the document is the owner?

Reply:
Yes, see https://github.com/decentralized-identity/ethr-did-resolver/blob/master/src/__tests__/ethr-did-resolver-test.js#L172-L175 It is the controller's `ethereumAddress`

Reply:
Great! That makes sense, I'll use that and remove the owner prop.

The words 'owner', 'controller', and 'delegate' are a bit confusing here and in the figma. Here you say 'set controller' but are talking about the owner. In the figma, controller and delegate are used interchangeably. Which is correct?